src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -208,20 +233,24 @@ bool emberAfGroupsClusterViewGroupCallback(app::CommandHandler * commandObj, con
         return true;
```

Comment:
Is there that we could add `IsUnicast()` to `CommandHandler` ? This would cover insuring we do not attempt to process multicast view-group.

Actually, I am curious how we can even determine which commands can be sent to groups. Very few command are applicable to groups...

We need to make sure, for instance, that there are no Manage or Administer-privilege commands ever processed for groups. For commands that require a response, they also cannot be processed, but that's a separate layer.

@bzbarsky-apple @mlepage-google This is a corner of access control we need to consider.

Reply:
Cam has an open issue to add a "must be multicast" or "must be unicast" quality for commands in the spec.

But it sounds like you want this to apply much more broadly than just the small set we have identified so far that must be unicast?